javascript/src/js/requests.txt

```diff
@@ -0,0 +1,9 @@
+/json/rdm/section_info
```

Comment:
removed it and put it as a comment inside of the app.js where it was more relevant